### PR TITLE
252-added-in-app-review

### PR DESCRIPTION
**Pull request by omen273**

### CODE REVIEW

File: app/src/main/java/com/omen273/crossLingo/GameActivity.kt [hunk 1]
```diff
@@ -358,9 +359,34 @@ class GameActivity : AppCompatActivity(), CrosswordView.OnLongPressListener,
...
+            }
+        }
+    }
+
     override fun onCrosswordSolved(view: CrosswordView) {
         star_number.text = (star_number.text.toString().toInt() + BONUS_ON_SOLVE).toString()
-        showFinishGameDialog()
+        val path = File(filesDir, "solved_crossword_number.json")
+        if (path.exists()) {
+            val number = FileInputStream(path).use { ConfigReader().solvedCrosswordNumber(it) }
```
Comment (greenEkatherine) [inline app/src/main/java/com/omen273/crossLingo/GameActivity.kt:381]: what happens if file is empty or corrupted? may be add exception handling?

File: app/src/main/java/com/omen273/crossLingo/GameActivity.kt [hunk 2]
```diff
@@ -358,9 +359,34 @@ class GameActivity : AppCompatActivity(), CrosswordView.OnLongPressListener,
...
+            if (number == 3 || number % 100 == 0) requestReview()
```
Comment (greenEkatherine) [inline app/src/main/java/com/omen273/crossLingo/GameActivity.kt:382]: I thought we agreed on 4

**Review by greenEkatherine — CHANGES_REQUESTED** [review]
Please address my comments

File: app/src/main/java/com/omen273/crossLingo/GameActivity.kt (hunk 2 above)
Comment (omen273) [reply app/src/main/java/com/omen273/crossLingo/GameActivity.kt:382]: It is 4

**Review by greenEkatherine — APPROVED** [review]

**Pull request merged**